src/main/java/org/jenkinsci/plugins/github/webhook/GHEventHeader.java

```diff
@@ -42,7 +42,9 @@
          * @return parsed {@link GHEvent} or null on empty header or unknown value
          */
         @Override
-        public Object parse(StaplerRequest request, GHEventHeader a, Class type, String parameterName) throws ServletException {
+        public Object parse(StaplerRequest request,
+                            GHEventHeader a,
+                            Class type, String parameterName) throws ServletException {
```

Comment:
don't follow logic, 3 args on new line, one not. Just put new line before throws